test/ERC20Permit.js

```diff
@@ -27,9 +27,7 @@ describe('ERC20 Permit', () => {
         owner = wallets[0];
         user = wallets[1];
 
-        interchainToken = await deployContract(owner, 'InterchainToken', [owner.address]);
-        interchainTokenDeployer = await deployContract(owner, 'InterchainTokenDeployer', [interchainToken.address]);
-
+        ({ interchainTokenDeployer } = await deployAll(owner, 'Test'));
```

Comment:
if you make the previous suggested change, then you don't need to deploy all contracts for this to work